import-service/import/import-products-file.ts

```diff
@@ -0,0 +1,28 @@
+import * as AWSLambda from "aws-lambda";
+import * as AWSSDK from "aws-sdk";
+
+export const importProductsFile = async (event: AWSLambda.APIGatewayEvent) => {
+  const s3 = new AWSSDK.S3({ region: "eu-west-1", signatureVersion: "v4" });
+  try {
+    const { name } = event.queryStringParameters;
+    const params = { Bucket: "app-import-storage", Key: `uploaded/${name}`, Expires: 60, ContentType: "text/csv" };
+
+    return new Promise((res, rej) => {
+      s3.getSignedUrl("putObject", params, (err, url) => {
```

Comment:
Just FYI. There is s3.getSignedUrlPromise method, you can use it in order to avoid creating promise by yourself.

Reply:
WOW! thank you!